app/src/controllers/testItem/selectors.js

```diff
@@ -49,6 +49,8 @@ const isListView = (query, namespace) => {
 export const isListViewSelector = (state, namespace) =>
   isListView(pagePropertiesSelector(state), namespace);
 
+const itemTitle = (item) => `${item.name} ${item.number ? `#${item.number}` : ''}`;
```

Comment:
`${item.name}${item.number && ` #${item.number}`}`;

Reply:
I see your point but in this case if the nested level has no number, then return an undefined. 
![image](https://user-images.githubusercontent.com/30735901/45823444-04965700-bcee-11e8-9cc3-1bd1f9113a09.png)
@ypetya has a good point: Is it possible that item.number == 0 ? If yes then probably we need an even better solution.

Reply:
I think a formatter function will be enough. I don't think that item.number can be 0, but it would be good to cover this case too.

Reply:
@DiscoElevator  Could you please check it again? Thank you.